cmd/server/iwf/iwf.go

```diff
@@ -124,7 +124,11 @@ func start(c *cli.Context) {
 		if err != nil {
 			rawLog.Fatalf("Unable to connect to Temporal because of error %v", err)
 		}
-		unifiedClient = temporalapi.NewTemporalClient(temporalClient, config.Interpreter.Temporal.Namespace, converter.GetDefaultDataConverter(), false)
+		retryPolicy := temporalapi.QueryWorkflowFailedRetryPolicy{
+			InitialIntervalSeconds: config.GetInitialIntervalSecondsWithDefault(),
+			MaximumAttempts:        config.GetMaximumAttemptsWithDefault(),
+		}
+		unifiedClient = temporalapi.NewTemporalClient(temporalClient, config.Interpreter.Temporal.Namespace, converter.GetDefaultDataConverter(), false, retryPolicy)
```

Comment:
If you have `config.GetInitialIntervalSecondsWithDefault()`, then the expectation is that the code that need to use the value will call it to get the value with default, instead of passing into here. 

So it will be `temporalapi.NewTemporalClient(temporalClient, config.Interpreter.Temporal.Namespace, converter.GetDefaultDataConverter(), false, config.ApiConfig.QueryRetryPolicy)`

Also consider using pointers to make this optional and load the default values in `client.go`.
Basically the config can be defined it as a pointer instead of struct. So that internally, we just need to pass a `nil` into this, which should let it use the default value.